### PR TITLE
Move home of crowbar user to /var/lib/crowbar

### PR DESCRIPTION
**Pull request by vuntz**

### CODE REVIEW

**Comment by rhafer** [comment]
Makes sense. +1

File: chef/cookbooks/crowbar/files/default/knife.rb [hunk 1]
```diff
@@ -1,2 +0,0 @@
-node_name "chef-webui"
```
Comment (aspiers) [inline chef/cookbooks/crowbar/files/default/knife.rb]: -1, this is still used by `cookbook_file "/root/.chef/knife.rb"` on line 178 of the `crowbar::default` recipe, which caused the gate to break.

**Comment by jsuchome** [comment]
+1

**Comment by tboerger** [comment]
And what happens to the knife config now?

**Comment by vuntz** [comment]
> And what happens to the knife config now?

Does the crowbar user use knife?

**Comment by tboerger** [comment]
> Does the crowbar user use knife?

Maybe just the administrator that signs in as crowbar user on the shell

**Comment by vuntz** [comment]
> Maybe just the administrator that signs in as crowbar user on the shell

The crowbar user has /sbin/nologin as shell, so no :-)

**Comment by tboerger** [comment]
Than it's fine :)
:+1:

**Pull request merged**